### ISSUE TITLE
Cannot mirror a custom repository

### ISSUE THREAD

**Issue by doccaz** [SUSE/rmt#490]
If I try to force mirroring for a custom repository, rmt-cli says it doesn't exist:

```
# rmt-cli repos custom list
...
| 1160 | SLE15_python                    | https://download.opensuse.org/repositories/devel:/languages:/python/SLE_15/                        | Not Mandatory | Mirror  |                    
...

# # rmt-cli mirror repository 1160
Repository with ID 1160 not found

```

It works if I use an ID from a regular repository. Shouldn't it mirror the custom repository too?
If I just run "rmt-cli mirror all", the custom repos are included.

**Comment by tmuntaner** [comment]
Hi @doccaz I made an alternate approach to this issue with #618. 

I saw that we could use the same technique we use for products and solve the scc_id vs id issue, while avoiding the need to create separate commands. Any feedback on that approach would be great.

Soon, the team will meet and I will present the different approaches for this issue. Then, we'll decide how to continue.

Thank you for your contribution.
